app/MyApplicationLoader.scala

```diff
@@ -0,0 +1,62 @@
+import configuration.Config
+import controllers._
+import filters.{AddEC2InstanceHeader, AddGuIdentityHeaders, CheckCacheHeadersFilter, HandleXFrameOptionsOverrideHeader}
+import loghandling.Logstash
+import monitoring.SentryLogging
+import play.api.ApplicationLoader.Context
+import play.api._
+import play.api.libs.ws.ahc.AhcWSComponents
+import play.api.mvc.EssentialFilter
+import play.filters.csrf.CSRFComponents
+import play.filters.headers.{SecurityHeadersConfig, SecurityHeadersFilter}
+import router.Routes
+import services.TouchpointBackends
+
+class MyApplicationLoader extends ApplicationLoader {
```

Comment:
https://www.playframework.com/documentation/2.6.x/ScalaCompileTimeDependencyInjection
"The application loader is constructed and invoked every time the application is reloaded, to load the application.

This trait’s load method takes as an argument the application loader Context, which contains all the components required by a Play application that outlive the application itself and cannot be constructed by the application itself."